package.json

```diff
@@ -15,12 +15,12 @@
     "prepublish": "in-publish && npm run build || not-in-publish",
     "postpublish": "npm run clean",
     "lint": "eslint lib bin/* test",
-    "pretest": "npm run lint",
-    "test": "npm run test:cover && npm run test:integration",
-    "test:integration": "npm run build && rimraf ./test/integration/tmp && BABEL_ENV=test babel-node ./test/runner-integration",
-    "test:cover": "BABEL_ENV=test babel-node ./node_modules/istanbul/lib/cli.js cover test/runner-unit",
-    "test:only": "BABEL_ENV=test babel-node ./test/runner-unit",
-    "test:debug": "BABEL_ENV=test babel-node debug ./test/runner-unit",
+    "pretest": "npm run lint && npm run build && rimraf ./test/integration/tmp",
+    "test": "npm run test:unit && npm run test:integration",
+    "test:unit": "jest --testPathPattern=test/unit --coverage",
+    "test:unit:debug": "npm run test:unit -- --watch",
```

Comment:
Does this replace the `node inspect` ?

Reply:
Hmm nope, it doesn't. I will try to find a solution.

Reply:
done :)